src/main/resources/pdl/hentperson-enkel.graphql

```diff
@@ -0,0 +1,41 @@
+query($ident: ID!) {person: hentPerson(ident: $ident) {
```

Comment:
Jeg synes det ser ut som du henter mye mer data enn vi trenger. Trenger vi alt dette for at systemet skal virke?

Reply:
ja, stemmer. fjernet unødvendige data fra respons. vi trenger bare fødselsdato, navn og kjønn fra responsen